static/js/tools/scatter/statvar.tsx

```diff
@@ -33,28 +33,21 @@ import {
   Label,
   Input,
 } from "reactstrap";
-import { Menu } from "../statvar_menu/menu";
-import { NoopStatsVarFilter, TimelineStatsVarFilter } from "../commons";
-import { StatVarNode, getStatVar } from "../statvar_menu/util";
-import {
-  Context,
-  EmptyAxis,
-  Axis,
-  AxisWrapper,
-  IsLoadingWrapper,
-} from "./context";
-import { NamedPlace } from "../../shared/types";
-import { nodeGetStatVar } from "./util";
+import { StatVarInfo, getStatVarInfo } from "../statvar_menu/util";
+import { Context, EmptyAxis, Axis, AxisWrapper } from "./context";
+import { StatVarHierarchyType } from "../../shared/types";
+import { StatVarHierarchy } from "../../stat_var_hierarchy/stat_var_hierarchy";
 
-interface NamedStatVar {
+const SAMPLE_SIZE = 3;
```

Comment:
add a short description for this - what is being sampled?

Reply:
done